src/lib/auth/authconf.rb

```diff
@@ -755,18 +755,99 @@ def aux_apply
                 service_disable_stop('nscd')
             end
         end
-        
+
+        def IsInstalledVersionNewerOrEqual(installed_rpm_version, test_rpm_version)
+            installed_rpm_version_l = Yast::Builtins.filter(
+                Yast::Builtins.splitstring(installed_rpm_version, "-.+")
```

Comment:
Please, use ruby std instead of `Yast::Builtins`, if possible:
```suggestion
                installed_rpm_version.split(/ [-.+]/)
```